File: db4on/instrumentation/CFNativeQueriesEnabler.Tests.Subject/Item.cs
/* Copyright (C) 2004 - 2006  db4objects Inc.   http://www.db4o.com */

namespace CFNativeQueriesEnabler.Tests.Subject
{
	public class Item
	{
		private string _name;

		public Item(string name)
		{
			_name = name;
		}

		public string Name
		{
			get { return _name; }
		}
	}
}

File: reference/src/com/db4odoc/commitcallbacks/Item.java
/* Copyright (C) 2007 db4objects Inc. http://www.db4o.com */

package com.db4odoc.commitcallbacks;


public class Item {
	private int _number;
	private String _word;
	
	public Item(int number, String word){
		_number = number;
		_word = word;
	}
	
	public String getWord(){
		return _word;
	}
	
	public int getNumber(){
		return _number;
	}
	
	public String toString(){
		return _number + "/" + _word;
	}
}
